src/git-istage/ConsoleCommand.cs

```diff
@@ -6,13 +6,24 @@ internal sealed class ConsoleCommand
     {
         private readonly Action _handler;
         private readonly ConsoleKey _key;
+        public readonly string Description;
```

Comment:
Should be named differently:

```suggestion
        public readonly string _description;
```

Reply:
I this in purpose because it's public field. I can change it to private and add a public `GetDescription` method.

Reply:
Oh, I see. In that case make it read-only property. Those can be assigned to from the constructor.